test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/DefaultEditorTemplatesTest.cs

```diff
@@ -827,7 +829,8 @@ public void Editor_InputTypeDateTime_RendersAsDateTime()
         [InlineData("datetime-local", null, "2000-01-02T03:04:05.060", "datetime-local")]
         [InlineData("DateTimeOffset", "{0:o}", "2000-01-02T03:04:05.060-05:00", "text")]
         [InlineData("time", "{0:t}", "03:04:05.060", "time")]
-        [InlineData("month", "{0:yyyy-MM}", "2000-01", "month")]
+        [InlineData("month", null, "2000-01", "month")]
+        [InlineData("week", null, "1999-W52", "week")]
```

Comment:
For `month` and `week`, test another `editFormatString` value. Make sure the user's explicit action is honored. (Test was previously using the default format string for `month` explicitly but use something else.)

Same for the three similar sets of `[InlineData(...)]` below.